examples/inline_osr.sou

```diff
@@ -0,0 +1,15 @@
+function main ()
```

Comment:
Or at the very least, a comment explaining which opts to run (since we won't be able to dig out the commit message).